lib/perl/Genome/Model/Tools/Capture/FormatSnvs.t

```diff
@@ -4,10 +4,56 @@ use strict;
 use warnings;
 
 use above 'Genome';
-use Test::More tests => 1;
+use Test::More;
 
-# This test was auto-generated because './Model/Tools/Capture/FormatSnvs.pm'
-# had no '.t' file beside it.  Please remove this test if you believe it was
-# created unnecessarily.  This is a bare minimum test that just compiles Perl
-# and the UR class.
 use_ok('Genome::Model::Tools::Capture::FormatSnvs');
+
+BEGIN {
+  $ENV{UR_DBI_NO_COMMIT} = 1;
+  $ENV{UR_USE_DUMMY_AUTOGENERATED_IDS} = 1;
+}
+
+my $expected_output_dir = $ENV{"GENOME_TEST_INPUTS"} . "Genome-Model-Tools-Capture-FormatSnvs/2015-02-12/";
```

Comment:
The fancy new way of doing this is:
```perl
use Genome::Utility::Test; #with the other "use" statements
Genome::Utility::Test->data_dir_ok('Genome::Model::Tools::Capture::FormatSnvs','2015-02-12');
```

It will assemble the directory path from the module name and version, and add a single ok() test that it exists.